engine/engine/src/engine.cpp

```diff
@@ -998,10 +996,6 @@ namespace dmEngine
         engine->m_MeshContext.m_Factory       = engine->m_Factory;
         engine->m_MeshContext.m_MaxMeshCount = dmConfigFile::GetInt(engine->m_Config, "mesh.max_count", 128);
 
-        engine->m_SpineModelContext.m_RenderContext = engine->m_RenderContext;
-        engine->m_SpineModelContext.m_Factory = engine->m_Factory;
-        engine->m_SpineModelContext.m_MaxSpineModelCount = max_spine_count;
-
```

Comment:
Another step towards removal, making the default implementation into an actual extension.